cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/naming/AbstractCgmesAliasNamingStrategy.java

```diff
@@ -32,9 +32,9 @@
  */
 public abstract class AbstractCgmesAliasNamingStrategy implements NamingStrategy {
 
-    private final BiMap<String, String> idByUuid = HashBiMap.create();
-    private final Map<String, String> uuidSeed = new HashMap<>();
-    private final NameBasedGenerator nameBasedGenerator;
+    public final BiMap<String, String> idByUuid = HashBiMap.create();
```

Comment:
I think it would be preferable to keep these attributes `private`. If you need to use them from subclasses, you can change the access to `protected` instead of `public`. If you need to obtain them from other classes, keep them `private` and increase this class adding a function returning them or (better) crate new methods that extend this class with the required functionality that use the private data without exposing it to the outside world.